Routing/Loader/Reader/RestActionReader.php

```diff
@@ -550,9 +613,9 @@ private function readRouteAnnotation(\ReflectionMethod $reflectionMethod)
     {
         $annotations = [];
 
-        foreach (['Route', 'Get', 'Post', 'Put', 'Patch', 'Delete', 'Link', 'Unlink', 'Head', 'Options'] as $annotationName) {
-            if ($annotations_new = $this->readMethodAnnotations($reflectionMethod, $annotationName)) {
-                $annotations = array_merge($annotations, $annotations_new);
+        foreach ($this->convertAvailableHttpMethodsToAnnotationNames() as $annotationName) {
```

Comment:
I do not understand why not just using:
```php
$annotations = $this->readMethodAnnotations($method, 'Route');
```
Thanks to inheritance, this should match all annotations.